### DISCUSSION TITLE
Can Metrics Server Support 2 Replicas?

### DISCUSSION THREAD

**Discussion by penghuazhou** [kedacore/keda#3705] [Q&A / Need Help] [answered]
keda: v2.8.1 

Can Metrics Server Support 2 Replicas?

<img width="986" alt="image" src="https://user-images.githubusercontent.com/13001038/192996292-e14e38a7-d38d-4f7a-b6a6-38be0f149149.png">

**Comment by JorTurFer** [comment, marked as answer]
Hey,
Metrics server supports multiple replicas, you can scale it if you want. The problem, as the picture says, is that for balancing the load you need to modify the node configuration, and that isn't allowed in cloud providers, but for HA purposes, you could have more than 1 instance.
The chart has this parameter for that `metricsServer.replicaCount`

**Reply by penghuazhou** [reply]
thanks
